src/config.ts

```diff
@@ -1,23 +1,30 @@
-const DB_NAME = process.env.DB_NAME || 'volta'
-const DB_USERNAME = process.env.DB_USERNAME || 'postgres'
-const DB_PASSWORD = process.env.DB_PASSWORD
-const DB_ADDRESS = process.env.DB_ADDRESS || 'local-pg'
-const DB_PORT = process.env.DB_PORT || 5432
+import epsagon from 'epsagon'
+
+const {
+  DB_NAME = 'volta',
+  DB_USERNAME = 'postgres',
+  DB_PASSWORD = '',
+  DB_HOST = 'local',
+  DB_PORT = 5432,
+  EPSAGON_APP_TOKEN,
+  EPSAGON_APP_NAME,
+} = process.env
+
+const DB_ENDPOINT = encodeURI(`postgresql://${encodeURIComponent(DB_USERNAME)}:${encodeURIComponent(DB_PASSWORD)}@${encodeURIComponent(DB_HOST)}:${DB_PORT}/${DB_NAME}`)
```

Comment:
I don't think you'll need to call `encodeURI` here since you're already encoding the db params. 
```suggestion
const DB_ENDPOINT = `postgresql://${encodeURIComponent(DB_USERNAME)}:${encodeURIComponent(DB_PASSWORD)}@${encodeURIComponent(DB_HOST)}:${DB_PORT}/${DB_NAME}`
```

Reply:
@whoops, good catch

Reply:
@ryanliszewski whoops good catch